src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3663,9 +3665,10 @@ void MethodContext::repGetThreadLocalStaticBlocksInfo(CORINFO_THREAD_STATIC_BLOC
 
     pInfo->tlsIndex.accessType = (InfoAccessType)value.tlsIndex.accessType;
     pInfo->tlsIndex.addr = (void*)value.tlsIndex.handle;
-    pInfo->offsetOfMaxThreadStaticBlocks = value.offsetOfMaxThreadStaticBlocks;
-    pInfo->offsetOfThreadLocalStoragePointer = value.offsetOfThreadLocalStoragePointer;
-    pInfo->offsetOfThreadStaticBlocks = value.offsetOfThreadStaticBlocks;
+    pInfo->offsetOfMaxThreadStaticBlocks    = value.offsetOfMaxThreadStaticBlocks;
```

Comment:
This function should also take `bool isGCType` and use that to determine the key.

Also, please align the `=`